tutorials/jdbcToJdbc/src/test/java/com/example/mydtapp/ApplicationTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Put your copyright and license info here.
+ */
+package com.example.mydtapp;
+
+import java.io.IOException;
+
+import javax.validation.ConstraintViolationException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.apache.hadoop.conf.Configuration;
+
+import com.datatorrent.api.LocalMode;
+
+/**
+ * Test the DAG declaration in local mode.
+ */
+public class ApplicationTest {
+
+  @Test
```

Comment:
Add a comment here about necessary preconditions for this test to work and add `@Ignore` annotation.